airflow/contrib/operators/gcp_spanner_operator.py

```diff
@@ -53,6 +54,9 @@ class CloudSpannerInstanceDeployOperator(BaseOperator):
                        'gcp_conn_id')
     # [END gcp_spanner_deploy_template_fields]
 
+    REQUIRED_ATTRIBUTES = ['instance_id']
```

Comment:
Is it realy required? `instnace_id` is required parameter in `__init__`. Python do this check.